article/_posts/2019-05-23-GAIL.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: review
+title:  "Generative Adversarial Imitation Learning"
+tags:   deep-learning reinforcement" 
```

Comment:
Spare double inverted comma.

Reply:
May be the `adversarial` tag could be added, as well as the `imitation-learning` tag (and be added to the `tags.yml` if not already present).

Reply:
Fixed in 01716a8